Document HTTP headers structure (#1645)
grafana/k6-docs: docs/sources/next/javascript-api/k6-http/response/_index.md

| `Response.error_code`                                                                                                           | number   | [Error code](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/error-codes) if there was a non-HTTP error or 4xx or 5xx HTTP error it will be set to a specific code that describes the error. (Added in 0.24.0)                                                                                                                                                                                                                                                                                                                                                                                                                 |

> | `Response.headers`                                                                                                              | object   | Key-value pairs representing all HTTP headers sent by the server. Note that the header names are in [canonical form](https://pkg.go.dev/net/http#CanonicalHeaderKey); for example, if the server responds with "accept-encoding", the key will be "Accept-Encoding". When requesting a header by a specific name, an array of strings is returned since the header can hold multiple values. In the case of a single-valued header, you can access it by index, such as `Response.headers["my_key"][0]`.                                                                                                  |

Reviewer: ```suggestion
| `Response.headers`                                                                                                              | object   | Key-value pairs representing all HTTP headers sent by the server. Note that the header names are in [canonical form](https://pkg.go.dev/net/http#CanonicalHeaderKey); for example, if the server responds with "accept-encoding", the key will be "Accept-Encoding". When requesting a header by a specific name, an array of strings is returned since the header can have multiple values. So you can access each value by its index - for the first value (or for single-valued header) that will be `Response.headers["my_key"][0]`.          

                                                                                                                                                                                |
```

> In the case of a single-valued header, you can access it by index, 

sounds like otherwise you can not - as if they were multiple values, you need to do something else.
